packages/utils/src/dhis.js

```diff
@@ -67,6 +70,8 @@ export const getDhisConfig = ({
   }
   const serverName = serverNameInput || getServerName(entityCode, isDataRegional);
   const serverUrl = getServerUrlFromName(serverName);
+  const serverSyncEnableParam = getServerVariable(serverName, 'DHIS_SYNC_ENABLE');
+  const serverSyncEnable = serverSyncEnableParam ? serverSyncEnableParam === 'true' : true;
```

Comment:
This will mean the default is `false`, so we'll need to add environment variables to turn it on for the existing servers. Would prefer to flip the default.

Perhaps the variable can be `serverReadOnly` instead, which flips everything?